src/cmr/concepts/collectionDraft.js

```diff
@@ -20,6 +20,8 @@ export default class CollectionDraft extends Concept {
    */
   constructor(headers, requestInfo, params) {
     super('collectionDraft', headers, requestInfo, params)
+    // Collection drafts do not support associations
+    delete this.setEssentialJsonValues
```

Comment:
This feels like a code smell. We should update the inheritance class rather than remove functions from an interface implementation.

Reply:
Yeah, I was conflicted on whether to do it this way or that way and went with this way because it seemed more explicit, but wasn't sure. I will change it